packages/admin-panel-server/src/connections/CentralConnection.ts

```diff
@@ -25,8 +25,8 @@ const translateParams = (queryParameters?: Record<string, unknown>) => {
 /**
  * @deprecated use @tupaia/api-client
  */
-export class MeditrakConnection extends ApiConnection {
-  public baseUrl = MEDITRAK_API_URL;
+export class CentralConnection extends ApiConnection {
```

Comment:
These 'connection' classes are probably my most irksome feeling about this rename. I feel like `CentralConnection` really tells me nothing. Bot on the other hand, I feel that we've sort of gone down that route by design, the server itself is really just a generic place. What we could do is create explicit APIs around it in the api-client package (say `CrudApi`, `AuthApi`, etc.)

Reply:
I like this idea. That could be an easier way to manage permission for each api.